specification/logstash/_types/Pipeline.ts

```diff
@@ -35,10 +35,35 @@ export class PipelineSettings {
   'queue.checkpoint.writes': integer
 }
 export class Pipeline {
+  /**
+   * Description of the pipeline.
+   * This description is not used by Elasticsearch or Logstash.
+   */
   description: string
+  /**
+   * Date the pipeline was last updated.
+   * Must be in the `yyyy-MM-dd'T'HH:mm:ss.SSSZZ` strict_date_time format.
+   */
   last_modified: DateTime
+  /**
+   * Optional metadata about the pipeline.
+   * May have any contents.
+   * This metadata is not generated or used by Elasticsearch or Logstash.
+   */
   pipeline_metadata: PipelineMetadata
+  /**
+   * User who last updated the pipeline.
+   */
   username: string
+  /**
+   * Configuration for the pipeline.
+   * @doc_id logstash-configuration-file-structure
+   */
   pipeline: string
+  /**
+   * Settings for the pipeline.
+   * Supports only flat keys in dot notation.
+   * @doc_id logstash-logstash-settings-file
+   */
   pipeline_settings: PipelineSettings
```

Comment:
I think it could be good to also document the pipeline settings (above, in the same file), with the information from https://www.elastic.co/guide/en/logstash/master/logstash-settings-file.html

Reply:
Thanks for the heads-up! Added in [f8e4f8a](https://github.com/elastic/elasticsearch-specification/pull/2240/commits/f8e4f8a5a812f805931ef370942ec8d4d1d1db48).